File: MultiBoxClientJavaCore/src/sk/hackcraft/multibox/model/LibraryItemType.java
package sk.hackcraft.multibox.model;

public enum LibraryItemType
{
	DIRECTORY(1),
	MULTIMEDIA(2);
	
	private final long id;
	
	private LibraryItemType(long id)
	{
		this.id = id;
	}
	
	public long getId()
	{
		return id;
	}
}

File: MultiboxClientCSharpCore/Model/LibraryItemType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;


namespace MultiboxClientCSharpCore.Model
{
	public enum LibraryItemType
	{
		Directory = 1,
		Multimedia = 2,
	}

	public class LibraryItemTypeParser
	{
		private static Dictionary<string, LibraryItemType> map;

		static LibraryItemTypeParser()
		{
			map = new Dictionary<string, LibraryItemType>();

			map["DIRECTORY"] = LibraryItemType.Directory;
			map["MULTIMEDIA"] = LibraryItemType.Multimedia;
		}

		public static LibraryItemType Parse(string name)
		{
			return map[name];
		}
	}
}
